app-android-xamarin/Moodis/Moodis/Feature/SignIn/SignInActivity.cs

```diff
@@ -105,6 +107,8 @@ private void InitButtonsAndInputs()
                     if (SignInViewModel.Authenticate(usernameInput.Text, passwordInput.Text))
                     {
                         SetResult(Result.Ok);
+                        var cameraActivity = new Intent(this, typeof(CameraActivity));
+                        StartActivity(cameraActivity);
```

Comment:
Maybe it would be better to move this call of `StartActivity` to MainActivity's `OnActivityResult` method, where it checks for requestCode and resultCode.(just basically replace the comment there)